src/app/migration/src/src/app/modules/core/components/breadcrumbs/breadcrumbs.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="ui margin-bottom-10 margin-top-15 breadCrumbsBackground" *ngIf="breadCrumbsData" >
+  <div class="ui grid">
+      <div class="one wide column"></div>
+      <div class="ten wide column">
+          <div class="breadcrumppadding ui padded breadcrumb">
+              <span *ngFor="let item of breadCrumbsData; last as isLast">
+                  <a *ngIf="!isLast" (click)="openLink(item.url)"  class="breadcrumb-fontColor section">{{item.label}}</a>
```

Comment:
for the last item in breadcrumbs array navigation should not happen

Reply:
'!isLast' makes sure the last breadcrumb is omitted.